src/Math/Double.cpp

```diff
@@ -6,6 +6,7 @@
  \see    Double.hpp
  */
 #include "Math/Double.hpp"
+#include <boost/lexical_cast.hpp>
```

Comment:
boost usage needs to be assessed on Windows.
Hopefully it will work, because it would be very useful for event managements for outputs.